sigh/spec/manager_spec.rb

```diff
@@ -11,7 +11,7 @@
       Sigh.config = FastlaneCore::Configuration.create(Sigh::Options.available_options, options)
 
       val = Sigh::Manager.start
-      expect(val).to eq(File.expand_path("./AppStore_com.krausefx.app.mobileprovision"))
+      expect(val).to eq(File.expand_path("./AppStore_uuid_com.krausefx.app.mobileprovision"))
```

Comment:
Yes, that's why this change would break existing setups. These tests must keep working without them being changed.